src/roswire/definitions/package.py

```diff
@@ -100,18 +100,29 @@ def paths(shell: dockerblade.Shell,
               files: dockerblade.FileSystem
               ) -> List[str]:
         """Parses :code:`ROS_PACKAGE_PATH` for a given shell."""
-        path_str = shell.environ('ROS_PACKAGE_PATH')
-        package_paths: List[str] = path_str.strip().split(':')
         paths: List[str] = []
-        for path in package_paths:
-            try:
-                all_packages = files.find(path, 'package.xml')
-            except dockerblade.exceptions.DockerBladeException:
-                logger.warning('unable to find directory in ROS_PACKAGE_PATH:'
-                               f' {path}')
-                continue
-            package_dirs = [os.path.dirname(p) for p in all_packages]
+        if files.exists('/opt/ros/dashing/'):
```

Comment:
There's a couple of things to improve here:
(1) this is hardcoded for our particular Docker images, and only for Dashing-based ROS2 images (i.e., it won't work for Foxy or Eloquent). (2) this appears to ignore any and all of the colon workspaces (e.g., `/ros_ws/src`).